.dockerignore

```diff
@@ -19,6 +19,10 @@ _artifacts
 Makefile
 **/Makefile
 
+# We need the following yaml files while building clusterctl in the container
+!cmd/clusterctl/config/manifest/clusterctl-api.yaml
+!cmd/clusterctl/client/cluster/assets/cert-manager-test-resources.yaml
+
```

Comment:
Q: is this required?
This change doesn't seem related to this PR...

Reply:
This was done to fix this issue: https://github.com/kubernetes-sigs/cluster-api/pull/6405#issuecomment-1100418420

Reply:
Sounds good to me